docker/configure_workers_and_start.py

```diff
@@ -547,7 +549,7 @@ def generate_worker_log_config(
     return log_config_filepath
 
 
-def main(args, environ):
+def main(args: List[str], environ: MutableMapping[str, str]) -> None:
```

Comment:
I can't figure out why `environ` needs to be mutable?

Reply:
We mutate the `environ`, see e.g.g +559; I was trying to be a light touch here and avoid logic/behaviour changes.

(I was surprised to learn that the `os.environ` mapping is mutable!)

Reply:
Ah, I missed that!